### PR TITLE
javax.inject.Named references removed from v5

### PR DESCRIPTION
**Pull request by rnewbigging**
## What Has Changed?

Removed references of java.inject.Named from v5 OData Generator documentation.

### CODE REVIEW

**Comment by rnewbigging** [comment]
> Could you please add a section / a few sentences to the migration guide?

Would that be the "Update to Version 5" doc? I'd assume under "Removed APIs" or would you prefer a new section?

**Comment by CharlesDuboisSAP** [comment]
You can write a new section under the "Update to Version 5" doc

File: docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx [hunk 1]
```diff
@@ -154,6 +154,15 @@ We will publish the full migration guide when we release v5.
   </table>
 </details>
 
+## List of changes to the OData Generator
+
+<details>
+    <summary>Full list of OData Generator changes</summary>
+
+- `javax.inject.Named` annotation removed from the OData Generator v2 and v4
```
Comment (Johannes-Schneider) [inline docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx:162]:
```suggestion
- `javax.inject.Named` annotation removed from the OData v2 and v4 generators
```

File: docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx [hunk 2]
```diff
@@ -154,6 +154,15 @@ We will publish the full migration guide when we release v5.
   </table>
 </details>
 
+## List of changes to the OData Generator
+
+<details>
+    <summary>Full list of OData Generator changes</summary>
+
+- `javax.inject.Named` annotation removed from the OData v2 and v4 generators
```
Comment (Johannes-Schneider) [inline docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx:162]:
```suggestion
- `javax.inject.Named` annotation removed from code generated by the OData v2 and v4 generators
```

File: docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx [hunk 3]
```diff
@@ -154,6 +154,15 @@ We will publish the full migration guide when we release v5.
   </table>
 </details>
 
+## List of changes to the OData Generator
+
+<details>
+    <summary>Full list of OData Generator changes</summary>
+
+- `javax.inject.Named` annotation removed from code generated by the OData v2 and v4 generators
```
Comment (Johannes-Schneider) [inline docs-java_versioned_docs/version-v5/guides/5.0-upgrade.mdx:162]: Actually, let's be a bit more precise even 😅

```suggestion
- `javax.inject.Named` annotation removed from code generated by the OData v2 and v4 generators. With that, [the `javax.inject` dependency](https://search.maven.org/artifact/javax.inject/javax.inject) is no longer needed to compile the generated code.
```

**Review by Johannes-Schneider — APPROVED** [review]

**Pull request merged**